core/types/calls.cc

```diff
@@ -1433,7 +1433,11 @@ class Magic_callWithSplat : public IntrinsicMethod {
 
 public:
     void apply(const GlobalState &gs, DispatchArgs args, const Type *thisType, DispatchResult &res) const override {
-        if (args.args.size() != 3) {
+        // args[0] is the receiver
+        // args[1] is true if private calls are ok
+        // args[2] is the method
+        // args[3] are the splat arguments
+        if (args.args.size() != 4) {
```

Comment:
Totally non-blocking:

if you wanted to, you could assign all these special numbers to local variable names, and use those names instead of the literal numbers down below (so that you'd have like `args.args[splatArgsIdx]` instead of `args.args[3]`.

Might read a little bit better.